### PR TITLE
Add the Nextcloud VM scripts to Linux installation

### PR DESCRIPTION
**Pull request by enoch85**
Self explanatory

cc @nextcloud/documentation

### CODE REVIEW

File: admin_manual/installation/source_installation.rst [hunk 1]
```diff
@@ -15,6 +15,8 @@ archive <https://nextcloud.com/install/>`_.
 .. note:: Admins of SELinux-enabled distributions such as CentOS, Fedora, and
    Red Hat Enterprise Linux may need to set new rules to enable installing
    Nextcloud. See :ref:`selinux_tips_label` for a suggested configuration.
+   
+You can also use the `Nextcloud VM scripts <https://github.com/nextcloud/vm/>`_ to install directly on a clean Ubuntu Server. It will setup everything for you and include scripts for automated installtion of apps like; Collabora, OnlyOffice, Talk and so on.
```
Comment (MorrisJobke) [inline admin_manual/installation/source_installation.rst]: Typo fixed in 7d050158fc7771a3f1ecaf136183baa2f9b63f11

Comment (enoch85) [reply admin_manual/installation/source_installation.rst]: Thanks!

**Pull request merged**